app/static/src/app/components/options/SensitivityOptions.vue

```diff
@@ -5,18 +5,25 @@
         <button class="btn btn-primary mb-4 float-end" @click="toggleEdit(true)">Edit</button>
         <ul>
           <li><strong>Parameter:</strong> {{settings.parameterToVary}}</li>
-          <li><strong>Scale Type:</strong> {{ settings.scaleType }}</li>
+          <li v-if="settings.variationType !== 'Custom'"><strong>Scale Type:</strong> {{ settings.scaleType }}</li>
```

Comment:
I feel this should now move *below* the variation type because then the dependency among the options is easier to work with

Reply:
Ok, so scale type below variation type, as it isn't relevant for custom variation type.